.travis.yml

```diff
@@ -4,8 +4,8 @@ dart:
   - dev
 
 dart_task:
-  - test:
-  - dartfmt:
+  - test
+  - dartfmt
```

Comment:
In this way we see the task name rather than "Dart: dev Task: [object Object]"

Reply:
Hmm, apparently that disabled formatting (but I've seen this used in other packages). Reverting.